.github/workflows/build.yml

```diff
@@ -0,0 +1,41 @@
+name: Build (and release)
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+  release:
+    types: [ "published", "edited" ]
```

Comment:
Can we please add the tag event as well? Right now, I prefer using a simple tag to release the package. I don't want to maintain the release notes manually, but of course, we can integrate something like [Release Drafter](https://github.com/release-drafter/release-drafter) as a follow-up. WDYT?

Reply:
Sure, I can add this event. FYI you don't need to write the relase notes manually. There's a "Generate release notes" button on the "Draft new release" page in GH.

Reply:
I didn't know about that feature 😎. Does it use the PR titles from the diff? That would be all we need, I guess. Then we can keep it as it is, and I'll try it that way next time.

Reply:
@HofmeisterAn Is this PR fine?